### ISSUE TITLE
feat: iOS 15 support

### ISSUE THREAD

**Issue by liamdebeasi** [ionic-team/ionic-framework#23423]
Tracking iOS 15 design changes here

**Comment by mhartington** [comment]
_Liam..._

**Comment by EinfachHans** [comment]
Wow, they really reverted the time picker? 😅😫

**Comment by liamdebeasi** [comment]
Just as I finished work on it in Ionic Framework too 🤦

**Comment by farisfaisalthena** [comment]
Maybe wait for a few betas as apple is known to change this during the beta. Hopefully they will stick with the current date picker

**Comment by BerkeAras** [comment]
The select style (ion-select) has changed to a kind of popover (which i mentioned in #21421)

**Comment by farisfaisalthena** [comment]
@liamdebeasi here’s a few things you can consider to add for ios maybe for the V6 :) Would update it here if theres more and also for any new changes with iOS 15

1. Press and hold and a contextual menu will appear similar like @BerkeAras  issue in #21421
2. Maybe reconsider again for the new toast for ios #22676 as it appears to be used not only for privacy focused and also i see some ios developers implementing this new style.

**Comment by liamdebeasi** [comment]
> Maybe wait for a few betas as apple is known to change this during the beta. Hopefully they will stick with the current date picker

Yes, we do not plan on addressing this time picker change right away to give Apple a chance to refine iOS 15.

> Press and hold and a contextual menu will appear similar like @BerkeAras issue in #21421

We don't have plans to implement https://github.com/ionic-team/ionic-framework/issues/21421 as a separate component at the moment. Popovers will have expanded positioning/alignment functionality in v6, and we will be evaluating additional animation/transitions.

> Maybe reconsider again for the new toast for ios #22676 as it appears to be used not only for privacy focused and also i see some ios developers implementing this new style.

Can you give me some examples of developers implementing the new style in non-privacy focused ways? As far as I am aware, developers do not have direct access to that API.

**Comment by cmaas** [comment]
Time picker doesn't seem to work in my Capacitor app at all with the current beta and unrealiable at best when deployed as a web app. Let's see what they'll do :)

**Comment by farisfaisalthena** [comment]
> Yes, we do not plan on addressing this time picker change right away to give Apple a chance to refine iOS 15.

Just curious will V6 ship with iOS 15 Design or we have to wait for V7?

> Can you give me some examples of developers implementing the new style in non-privacy focused ways? As far as I am aware, developers do not have direct access to that API.

This was taken from an app called 'Aviary' a twitter client app

![toast](https://user-images.githubusercontent.com/61012396/121188121-61ba0880-c89b-11eb-8391-523bfdcc321b.png)

**Comment by liamdebeasi** [comment]
> Just curious will V6 ship with iOS 15 Design or we have to wait for V7?

More to announce on this at Ioniconf (https://ionic.io/ioniconf), but you will not have to wait until v7 for iOS 15 updates.

--------

Going to lock this issue for now, but we will update the original post as we find more iOS 15-specific changes. Thanks!
